bindings/java/native/src/classes/address.rs

```diff
@@ -40,3 +50,67 @@ impl Address {
         self.address.address().clone()
     }
 }
+
+pub struct AddressBuilder {
+    builder: Rc<RefCell<Option<AddressBuilderRust>>>,
```

Comment:
I don't think it's wise to construct a builder pattern like this, as `RefCell` basically defers borrow checking to runtime.

See https://rust-unofficial.github.io/patterns/patterns/creational/builder.html for a reasonable structure of a builder.

Reply:
This is the only way we can have a builder pattern through java as far as i could find. If you have suggestions please let me know :) But all calls must be through references (&self, &mut self).